debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/JavaVersionHelper.java

```diff
@@ -25,30 +25,43 @@
 
 import com.ibm.j9ddr.CorruptDataException;
 import com.ibm.j9ddr.vm29.pointer.generated.J9JavaVMPointer;
-import com.ibm.j9ddr.vm29.types.UDATA;
 
 /**
  * JavaVersionHelper helps check if the JVM version is new enough for the modularity DDR commands
  */
 public class JavaVersionHelper 
 {
-	public final static int J2SE_SERVICE_RELEASE_MASK = 0xFFFF;
-	public final static int J2SE_19 = 9;
-	public final static int J2SE_JAVA_SPEC_VERSION_SHIFT = 8;
+	private final static int J2SE_SERVICE_RELEASE_MASK = 0xFFFF;
+	private final static int J2SE_JAVA_SPEC_VERSION_SHIFT = 8;
 	
 	/**
 	 * Returns true if the Java version is Java9 and up.
 	 * @param vm J9JavaVMPointer
 	 * @param out The output print stream
 	 * @return if the Java version is Java9 and up or not.
 	 * @throws CorruptDataException
+	 * @deprecated Use {@link #ensureMinimumJavaVersion(int, J9JavaVMPointer, PrintStream)} instead.
```

Comment:
These javadoc tags should not start with a capital, nor end with a period. Likewise for `ensureMinimumJavaVersion()`.